hack/addons/mindthegap-helm-registry/Dockerfile

```diff
@@ -1,19 +1,19 @@
-ARG MINDTHEGAP_VERSION=v1.14.4
```

Comment:
nit: can we rename this file since its no longer using `mindthegap`.
Also should it be `repository` instead registry to match the helm values?

Reply:
Done!